<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(1,35,38);</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>5</y>
     <width>1000</width>
     <height>550</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>나눔고딕 ExtraBold</family>
    </font>
   </property>
   <property name="toolTipDuration">
    <number>-3</number>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(1,35,38);
gridline-color: rgb(1,35,38);
color: rgb(1,35,38);
border-color: rgb(25, 25, 25);
gridline-color: rgb(30, 30, 30);</string>
   </property>
   <property name="tabPosition">
    <enum>QTabWidget::North</enum>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Rounded</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="iconSize">
    <size>
     <width>200</width>
     <height>17</height>
    </size>
   </property>
   <property name="elideMode">
    <enum>Qt::ElideMiddle</enum>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Overview</string>
    </attribute>
    <widget class="QTableWidget" name="map">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>70</y>
       <width>431</width>
       <height>381</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
border-radius:10px;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>110</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>나눔고딕 ExtraBold</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(82,242,226);</string>
     </property>
     <property name="text">
      <string>프로젝트 정보</string>
     </property>
    </widget>
    <widget class="QListView" name="listView">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>70</y>
       <width>221</width>
       <height>271</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255,255,255);
background-color: rgb(1,35,40);
border: 7px double rgb(82,242,226);
border-radius: 10px;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>800</x>
       <y>110</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>나눔고딕 ExtraBold</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(82,242,226);</string>
     </property>
     <property name="text">
      <string>속성별 색상</string>
     </property>
    </widget>
    <widget class="QListView" name="listView_2">
     <property name="geometry">
      <rect>
       <x>740</x>
       <y>70</y>
       <width>221</width>
       <height>271</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255,255,255);
background-color: rgb(1,35,40);
border: 7px double rgb(82,242,226);
border-radius: 10px;</string>
     </property>
    </widget>
    <widget class="QLabel" name="color_charge">
     <property name="geometry">
      <rect>
       <x>1400</x>
       <y>90</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="color_ws">
     <property name="geometry">
      <rect>
       <x>1400</x>
       <y>150</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="color_chute">
     <property name="geometry">
      <rect>
       <x>1400</x>
       <y>120</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="color_buffer">
     <property name="geometry">
      <rect>
       <x>1400</x>
       <y>180</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="color_block">
     <property name="geometry">
      <rect>
       <x>1400</x>
       <y>210</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>150</y>
       <width>180</width>
       <height>161</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="1">
       <widget class="QLineEdit" name="distributor">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>25</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(1,35,40);
color: rgb(107,242,242);
border: 1px solid rgb(107,242,242);</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <family>나눔고딕 ExtraBold</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(82,242,226);</string>
        </property>
        <property name="text">
         <string>Distributor</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <family>나눔고딕 ExtraBold</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(82,242,226);</string>
        </property>
        <property name="text">
         <string>Project ID</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="customer">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>25</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(1,35,40);
color: rgb(107,242,242);
border: 1px solid rgb(107,242,242);</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="projectid">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>25</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(1,35,40);
color: rgb(107,242,242);
border: 1px solid rgb(107,242,242);
</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_5">
        <property name="font">
         <font>
          <family>나눔고딕 ExtraBold</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(82,242,226);</string>
        </property>
        <property name="text">
         <string>Customer</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_6">
        <property name="font">
         <font>
          <family>나눔고딕 ExtraBold</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(82,242,226);</string>
        </property>
        <property name="text">
         <string>Center Name</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="centername">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>25</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(1,35,40);
color: rgb(107,242,242);
border: 1px solid rgb(107,242,242);</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="gridLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>780</x>
       <y>140</y>
       <width>160</width>
       <height>176</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_2"/>
    </widget>
    <widget class="QPushButton" name="c_charge">
     <property name="geometry">
      <rect>
       <x>892</x>
       <y>141</y>
       <width>37</width>
       <height>30</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>771</x>
       <y>285</y>
       <width>115</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>나눔고딕 ExtraBold</family>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(82,242,226);</string>
     </property>
     <property name="text">
      <string>블락</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>771</x>
       <y>213</y>
       <width>115</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>나눔고딕 ExtraBold</family>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(82,242,226);</string>
     </property>
     <property name="text">
      <string>워크 스테이션</string>
     </property>
    </widget>
    <widget class="QPushButton" name="c_ws">
     <property name="geometry">
      <rect>
       <x>892</x>
       <y>213</y>
       <width>37</width>
       <height>30</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>771</x>
       <y>249</y>
       <width>115</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>나눔고딕 ExtraBold</family>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(82,242,226);</string>
     </property>
     <property name="text">
      <string>버퍼</string>
     </property>
    </widget>
    <widget class="QPushButton" name="c_block">
     <property name="geometry">
      <rect>
       <x>892</x>
       <y>285</y>
       <width>37</width>
       <height>30</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="c_chute">
     <property name="geometry">
      <rect>
       <x>892</x>
       <y>177</y>
       <width>37</width>
       <height>30</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="c_buffer">
     <property name="geometry">
      <rect>
       <x>892</x>
       <y>249</y>
       <width>37</width>
       <height>30</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>771</x>
       <y>141</y>
       <width>115</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>나눔고딕 ExtraBold</family>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(82,242,226);</string>
     </property>
     <property name="text">
      <string>충전</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>771</x>
       <y>177</y>
       <width>115</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>나눔고딕 ExtraBold</family>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(82,242,226);</string>
     </property>
     <property name="text">
      <string>슈트</string>
     </property>
    </widget>
    <zorder>listView_2</zorder>
    <zorder>listView</zorder>
    <zorder>map</zorder>
    <zorder>label</zorder>
    <zorder>label_2</zorder>
    <zorder>color_charge</zorder>
    <zorder>color_ws</zorder>
    <zorder>color_chute</zorder>
    <zorder>color_buffer</zorder>
    <zorder>color_block</zorder>
    <zorder>gridLayoutWidget</zorder>
    <zorder>gridLayoutWidget_2</zorder>
    <zorder>c_charge</zorder>
    <zorder>label_11</zorder>
    <zorder>label_9</zorder>
    <zorder>c_ws</zorder>
    <zorder>label_10</zorder>
    <zorder>c_block</zorder>
    <zorder>c_chute</zorder>
    <zorder>c_buffer</zorder>
    <zorder>label_7</zorder>
    <zorder>label_8</zorder>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
